server/server_test.go

```diff
@@ -221,6 +222,49 @@ func (dbt *DBTest) mustQueryRows(query string, args ...interface{}) {
 	rows.Close()
 }
 
+// DBTestWithT like DBTest, but replace *check.C to *testing.T
+// TODO remove/rename when migrate server_test.go and statistics_handler_test.go to testify done
+type DBTestWithT struct {
```

Comment:
I think it can be abstract to the testkit package. Also, I remembered that there are https://github.com/jmoiron/sqlx.